docker/dev_vpp_agent/build-agent.sh

```diff
@@ -19,7 +19,7 @@ curl https://glide.sh/get | sh
 # checkout agent code
 mkdir -p $GOPATH/src/github.com/ligato
 cd $GOPATH/src/github.com/ligato
-git clone https://github.com/ligato/vpp-agent
+git clone https://github.com/mpundlik/vpp-agent
```

Comment:
Why did you need this committed to this pull request?

Reply:
not needed, removed